### ISSUE TITLE
BUG: Set maxheight for ui error with long stacktrace

### ISSUE THREAD

**Issue by mhsdesign** [neos/neos-ui#3622]
the error boundary was introduced with https://github.com/neos/neos-ui/pull/3485 

Todo add screenshot what went wrong 😅

**Comment by mhsdesign** [comment]
with #3996 the `FatalErrorView` uses our `ErrorView` which has better styling and should behave correctly

see aa89379c79a24734b9d62bbeb044bc5516234735
